pkg/archive/archive.go

```diff
@@ -322,6 +334,31 @@ func FileInfoHeader(name string, fi os.FileInfo, link string) (*tar.Header, erro
 	return hdr, nil
 }
 
+// fillGo18FileTypeBits fills type bits which have been removed on Go 1.9 archive/tar
```

Comment:
If it were to have a tag it would have to have go1.9 tag, but even then it's not needed since in go1.8 these bits are already set, so it would just set it again (it's OR'd)